packages/compare-slider-block/src/CompareSliderBlock.tsx

```diff
@@ -0,0 +1,397 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { FC, useEffect, useRef, useState } from 'react';
+import { ReactCompareSlider } from 'react-compare-slider';
+
+import { BlockProps } from '@frontify/guideline-blocks-settings';
+import {
+    AssetChooserObjectType,
+    FileExtensionSets,
+    useAssetUpload,
+    useBlockAssets,
+    useBlockSettings,
+    useEditorState,
+    useFileInput,
+} from '@frontify/app-bridge';
+import { radiusStyleMap, toRgbaString, useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
+
+import {
+    Alignment,
+    BlockSettings,
+    Height,
+    InheritSettings,
+    SliderImageSlot,
+    heightMap,
+    horizontalLabelPlacementStyleMap,
+    slotAssetSettingMap,
+    verticalLabelPlacementStyleMap,
+} from './types';
+import { Label } from './components/Label/Label';
+import { Strikethrough } from './components/StrikeThrough/Strikethrough';
+import { StrikethroughWrapper } from './components/StrikeThrough/StrikethroughWrapper';
+import { DEFAULT_STRIKETHROUGH_COLOR } from './const';
+import { LabelWrapper } from './components/Label/LabelWrapper';
+import { SliderLine } from './components/SliderLine/SliderLine';
+import { UploadView } from './components/UploadView/UploadView';
+import { EditorOverlay } from './components/EditorOverlay/EditorOverlay';
+
+export const CompareSliderBlock: FC<BlockProps> = ({ appBridge }) => {
```

Comment:
```suggestion
export const CompareSliderBlock: FC<BlockProps> = ({ appBridge }): ReactElement => {
```

Reply:
mmm, can you briefly explain why is that necessary?

Reply:
Well its not necessary, but otherwise it gets exported as JSX.Element (which also would be fine), but if we define it as ReactElement its more specific to the React ecosystem, and it includes additional type information like props, type and key